interface/VideoEncoderDefs.h

```diff
@@ -189,10 +189,14 @@ typedef struct VideoRateControlParams {
     uint32_t targetPercentage;
     uint32_t disableFrameSkip;
     uint32_t disableBitsStuffing;
+    uint32_t diffIPQP;// I/P frame QP difference
```

Comment:
Would better change the comment to "P frame qp  minus initQP"  to make it clearer.